website/docs/server-guides/advanced/reverse-proxy.md

```diff
@@ -144,3 +144,69 @@ server {
+                    <action type="Rewrite" url="http://192.168.1.100:3334/{R:1}" />
+                    <serverVariables>
+                        <set name="HTTP_X_ORIGINAL_ACCEPT_ENCODING" value="{HTTP_ACCEPT_ENCODING}" />
+                        <set name="HTTP_ACCEPT_ENCODING" value="" />
+                        <set name="HTTP_SEC_WEBSOCKET_EXTENSIONS" value="0" />
+                    </serverVariables>
+                </rule>
+            </rules>
+            <outboundRules>
+                <rule name="ReverseProxyOutboundRule1" preCondition="ResponseIsHtml1">
+                    <match filterByTags="A, Form, Img" pattern="^http(s)?://192.168.1.100:3334/(.*)" />
+                    <action type="Rewrite" value="http{R:1}://obico.mydomain.com/{R:2}" />
+                </rule>
+                <rule name="RestoreAcceptEncoding" preCondition="NeedsRestoringAcceptEncoding">
+                    <match serverVariable="HTTP_ACCEPT_ENCODING" pattern="^(.*)" />
+                    <action type="Rewrite" value="{HTTP_X_ORIGINAL_ACCEPT_ENCODING}" />
+                </rule>
+                <preConditions>
+                    <preCondition name="ResponseIsHtml1">
+                        <add input="{RESPONSE_CONTENT_TYPE}" pattern="^text/html" />
+                    </preCondition>
+                     <preCondition name="NeedsRestoringAcceptEncoding">
+                        <add input="{HTTP_X_ORIGINAL_ACCEPT_ENCODING}" pattern=".+" />
+                    </preCondition>
+                </preConditions>
+            </outboundRules>
+        </rewrite>
+    </system.webServer>
+</configuration>
+```
+
+You will also need to edit your `applicationHost.config` file (default location of \Windows\System32\inetsrv\config) to allow IIS to edit the necessary headers. Replace `Obico reverse proxy` with your IIS site name defintion.
+
+```
+     <location path="Obico reverse proxy">
+        <system.webServer>
+            <rewrite>
+                <allowedServerVariables>
+                    <add name="HTTP_ACCEPT_ENCODING" />
+                    <add name="HTTP_X_ORIGINAL_ACCEPT_ENCODING" />
+                    <add name="HTTP_SEC_WEBSOCKET_EXTENSIONS" />
+                </allowedServerVariables>
+            </rewrite>
+        </system.webServer>
+    </location>
+```
+
+Finally, on your Obico server, you will need to specify the fully qualified domain name for which you want Obico to issue cookies. There is likely some mechanism to make this work automatically (as Obico/Django should be able to detect the domain under which it is being accessed and refer to that value) but it is possible that the above IIS proxy configuration is not relaying the necessary informaiton for it to do that. A workaround is to manually specify the FQDN in `/backend/config/settings.py`. Add these two lines in the `SESSION_COOKIE` section near the beginning of the file, replacing `obico.mydomain.com` with your FQDN:
+
+```
```

Comment:
You mean users need to change the source code? I think a better way is to set them in .env and let them propogate into django. Otherwise most users probably won't know how to do it.

Reply:
Not the source code. Just the difference between editing:

`~.env`

and

~./backend/config/settings.py`

Certainly it would be fine to pull this value from the environment as well (as you're already overwriting other settings values). This is my first time with either Obico or Django though, so I don't know where to find 'overwrite these settings.py values from the environment' but if it's just a matter of appending the variable names onto an existing list, that's easily added to this PR.

Reply:
By `~./backend/config/settings.py`, do you mean `[obico-server-path]/backend/config/settings.py` ? If so, it's part of the source code.

Reply:
That's correct. 

I agree that an .env solution would be better, but that will have to come from someone who knows how to implement it.

Reply:
Ok... I guess I can merge your PR but I will add a warning that this is highly technical and it may prevent you from getting the future updates (git may refuse to update when you have modified the source code). not ideal but probably the best middle ground we can find

Reply:
Do we not know where the code that says 'pull these values from .env' lives because that's a django thing and not an obico thing?

I feel like it's got to be super simple, I just don't know where to look.